client/client.go

```diff
@@ -44,8 +52,11 @@ type Body struct {
 type Client struct {
 	apiKey      string
 	baseURL     string
-	client      *http.Client
+	orgName     string
+	client      *retryablehttp.Client
+	rateLimiter *rate.Limiter
 	contentType string
+	context     context.Context
```

Comment:
in general it's not correct to put contexts inside structs.  https://go.dev/blog/context-and-structs

Are you doing this for a particular reason?

Reply:
I am not, will remove :)